test/tools/utils.js

```diff
@@ -37,11 +35,6 @@ function readExampleFile(name, raw) {
         console.error(JSON.stringify(e));
```

Comment:
Should this one also be removed? Or is pre-existing to your work on NGSI-LD?

(Maybe a grep for `console.` in all the code base could help to find additional cases)

Reply:
Nope - that is a deliberate console output - see the previous line. It was part of the output of #832